yarn.lock

```diff
@@ -6597,6 +6637,34 @@ esbuild@~0.15.10:
     esbuild-windows-64 "0.15.18"
     esbuild-windows-arm64 "0.15.18"
 
+esbuild@~0.17.12:
```

Comment:
some changes from main are not merged yet, and we have older version of esbuild used in other places